docs/releasenotes/10.2.0.rst

```diff
@@ -105,3 +107,15 @@ Encoder errors now report error detail as string
 
 :py:exc:`OSError` exceptions from image encoders now include a textual description of
 the error instead of a numeric error code.
+
+Type hints
+^^^^^^^^^^
+
+Work has begun to add type annotations to Pillow, including:
+
+* :py:class:`~PIL.ContainerIO`
+* :py:class:`~PIL.FontFile` and subclasses
+* :py:class:`~PIL.ImageChops`
+* :py:class:`~PIL.ImageMode`
+* :py:class:`~PIL.ImageSequence`
+* :py:class:`~PIL.TarIO`
```

Comment:
It doesn't seem to affect the generated docs, but these are modules rather than classes.
```suggestion
* :py:mod:`~PIL.ContainerIO`
* :py:mod:`~PIL.FontFile` and subclasses
* :py:mod:`~PIL.ImageChops`
* :py:mod:`~PIL.ImageMode`
* :py:mod:`~PIL.ImageSequence`
* :py:mod:`~PIL.TarIO`
```